cartridge/failover.lua

```diff
@@ -369,6 +521,8 @@ local function cfg(clusterwide_config)
                 params.uri
             )
         elseif failover_cfg.state_provider == 'etcd2' then
+            -- TODO vclockkeeper handling in etcd2
```

Comment:
I don't see any issue for it

Reply:
Here it is https://github.com/tarantool/cartridge/issues/906